frontend/src/scenes/session-recordings/components/RecordingRow.tsx

```diff
@@ -0,0 +1,65 @@
+import clsx from 'clsx'
+import { useActions } from 'kea'
+import { IconPlayCircle } from 'lib/lemon-ui/icons'
+import { LemonSnack } from 'lib/lemon-ui/LemonSnack'
+import { ProfilePicture } from 'lib/lemon-ui/ProfilePicture'
+import { humanFriendlyDuration } from 'lib/utils'
+import { asDisplay } from 'scenes/persons/person-utils'
+import { sessionPlayerModalLogic } from 'scenes/session-recordings/player/modal/sessionPlayerModalLogic'
+import { sessionRecordingEventUsageLogic } from 'scenes/session-recordings/sessionRecordingEventUsageLogic'
+
+import { SessionRecordingType } from '~/types'
+
+import { ProjectHomePageCompactListItem } from '../../project-homepage/ProjectHomePageCompactListItem'
+
+export interface RecordingRowProps {
+    recording: SessionRecordingType
+}
+
+type ACTIVITY_DESCRIPTIONS = 'very low' | 'low' | 'medium' | 'high' | 'very high'
+
+function ActivityScoreLabel({ score }: { score: number | undefined }): JSX.Element {
+    const n = score ?? 0
+    let backgroundColor = 'bg-primary-alt-highlight'
+    let description: ACTIVITY_DESCRIPTIONS = 'very low'
+    if (n >= 90) {
+        backgroundColor = 'bg-success-highlight'
+        description = 'very high'
+    } else if (n >= 75) {
+        backgroundColor = 'bg-success-highlight'
+        description = 'high'
+    } else if (n >= 50) {
+        backgroundColor = 'bg-warning-highlight'
+        description = 'medium'
+    } else if (n >= 25) {
+        backgroundColor = 'bg-warning-highlight'
+        description = 'low'
+    }
+
+    return <LemonSnack className={clsx(backgroundColor, 'text-xs')}>activity: {description}</LemonSnack>
+}
+
+export function RecordingRow({ recording }: RecordingRowProps): JSX.Element {
```

Comment:
this was the watch next panel for the home page... 

we can't remove it because web analytics is using it

Reply:
Should web analytics use something else?

Reply:
no, something in here pointed me at fixing the watch next panel, which i realised wasn't used so i reorganised it

i meant that I could remove the watchnext panel but not the recording row... sorry wasn't meant to sound accusative!

Reply:
DW I didn't think you were accusing. Tone is hard over text!